README.md

```diff
@@ -1,7 +1,81 @@
 # Noir Benchmarks
-Benchmark `nargo prove` times of Noir programs spanning across different constraint counts (i.e. program sizes).
 
-## v0.26.0 Results
+Benchmarks for `nargo compile`, `nargo execute`, and `bb prove_ultra_honk` over
+Noir programs spanning across different constraint counts (i.e. program sizes).
+
+Results gathered with:
+
+- Nargo 1.0.0-beta.0
+- Barretenberg 0.63.0
+- Lenovo Thinkpad P16 Gen 2 with [13th Gen Intel(R) Core(TM)
+  i7-13700HX](https://www.intel.com/content/www/us/en/products/sku/232166/intel-core-i713700hx-processor-30m-cache-up-to-5-00-ghz/specifications.html)
+- Ubuntu Linux 23.04
+
+# Nargo v1.0.0-beta.0 and bb 0.63.0 Results
+
+![Results on Thinkpad P16 Gen 2](Thinkpad_P16_Gen2_v1.0.0-beta.0.png)
+
+| Primitive                    | Backend Circuit Size | Compile Time (s) | Execute Time (s) | Prove Time (s) |
+|------------------------------|:--------------------:|:----------------:|:----------------:|:--------------:|
+| keccak256_32B                |                20240 |            0.160 |            0.150 |          0.911 |
+| keccak256_32B_100_times      |              1733534 |            0.791 |            0.278 |         18.299 |
+| keccak256_532B               |                76817 |            0.252 |            0.168 |          1.429 |
+| keccak256_532B_10_times      |               747646 |            0.955 |            0.350 |          8.316 |
```

Comment:
To investigate:
Proving times look off, 747646 constraints should not be proved faster than 2^19 and 2^20 circuits

Reply:
Interesting! To make sure I understand, `keccak256_532B_10_times` takes 8.316 to prove, but according to the benchmarks (which I ran again and posted in #6,  the 2^19 circuit takes 10.331s.

Does `bb prove_ultra_honk` output the correct constraint count? This is how I got the number 747646:

```
$ time bb prove_ultra_honk -b ./target/keccak256_532B_10_times.json -w ./target/witness_keccak256_532B_10_times.gz -o ./proofs/keccak256_532B_10_times_bb_bin.proof 
Finalized circuit size: 747646
Log dyadic circuit size: 20

real    0m8.253s
user    0m51.111s
sys     0m1.932s
```

Reply:
There is `bb gates` command to get the gate count
`bb gates -b target/circuit.json | jq  '.functions[0].circuit_size'`

Reply:
Got the same number from `bb gates`, so yes that should be right.